Dockerfile

```diff
@@ -37,4 +37,5 @@ ENV ADDRESS="" \
     SETUP="false" \
     AUTO_UPDATE="true" \
     LOG_LEVEL="" \
-    BINARY_DIR="/app/config/bin"
+    BINARY_DIR="/app/bin" \
+    BINARY_STORE="/app/config/bin"
```

Comment:
Let's make it `BINARY_STORE_DIR` so it will reflect the new flag `--binary-store-dir` of the updater later.

Reply:
Added a commit to fix that